code/switchboard/client/read.client.en.ts

```diff
@@ -2,30 +2,29 @@ import {
   clusterApiUrl,
   Connection,
   Keypair,
-  PublicKey,
   LAMPORTS_PER_SOL,
 } from "@solana/web3.js";
-import * as switchboard from "@switchboard-xyz/switchboard-v2";
-async () => {
-  const payer = Keypair.generate();
-  const connection = new Connection(clusterApiUrl("devnet"));
-  const program = await switchboard.loadSwitchboardProgram(
-    "devnet",
-    connection,
-    payer
-  );
-  const airdropSignature = await connection.requestAirdrop(
-    payer.publicKey,
-    LAMPORTS_PER_SOL
-  );
+import { loadSwitchboardProgram, AggregatorAccount } from "@switchboard-xyz/switchboard-v2";
 
-  await connection.confirmTransaction(airdropSignature);
-  console.log("here");
-  const aggregatorAccount = new switchboard.AggregatorAccount({
-    program: program,
-    publicKey: aggregatorKey,
-  });
-  const result: any = await aggregatorAccount.getLatestValue();
+const main = async () => {
+  try {
+    const payer = Keypair.generate();
+    const connection = new Connection(clusterApiUrl("devnet"));
+    const program = await loadSwitchboardProgram("devnet", connection, payer);
+    const airdropSignature = await connection.requestAirdrop(
+      payer.publicKey,
+      LAMPORTS_PER_SOL
+    );
 
-  console.log(result.toNumber());
+    await connection.confirmTransaction(airdropSignature);
+    console.log("Airdrop confirmed");
+    const aggregatorAccount = new AggregatorAccount({
+      program: program,
+      publicKey: aggregatorKey,
+    });
+ const result = await aggregatorAccount.getLatestValue();
```

Comment:
You removed the `any` annotation here. If that was intentional then can you also update `read.client.preview.en.ts`?